utils/strings.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * capitalize first letter
+ * @param {string} text
+ */
+ export const capfl = text => {
+    return text.charAt(0).toUpperCase() + text.slice(1);
```

Comment:
? https://lodash.com/docs/4.17.15#upperFirst

Reply:
super, to mi vobec nenapadlo, ze lodash nieco take moze mat. mozeme teda tento PR povazovat za uzavrety, ja to upravim v `project-mobilonline-hu`

na druhej strane mi to ale dovolilo rozbehat tam jest a napisat si test 🙈